### PR TITLE
Added template tag in LicenseIcons.vue around the v-if img

### LINKED ISSUE #653: v-for and v-if on same node in LicenseIcons.vue
### Description
The IMG tag has both v-if and v-for on the same node so eslint is giving an error

### Screenshot
v-for-v-if-license

### Solution
we can add a template tag around, for the loop and the IMG will have to check only the condition

---
I would like to work on this

### PR DESCRIPTION
**Pull request by hemanth-hk**
Fixes #653

---

Initially, v-for and v-if are present on the same node. This `PR` corrects that and adds a template around to make it more readable and follows the style-guide of vue.

---

Developer Certificate of Origin Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open-source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

**Comment by hemanth-hk** [comment]
When I was writing the description my Grammarly pointed an error and a clarity check in the Developer Certificate of Origin, changes were 
1. change `open source` to `open-source` 
2. change `By making a contribution to this project` to `By contributing to this project`

I don't know where can I change them so I am commenting them here

**Comment by kgodey** [comment]
@hemanth-hk We're using the original DCO: https://elinux.org/Developer_Certificate_Of_Origin and will not be making changes, since it's a standard document.

File: src/components/LicenseIcons.vue [hunk 1]
```diff
@@ -4,12 +4,14 @@
      class="photo-license-icons"
      target="_blank"
      rel="noopener noreferrer">
-    <img class="photo-license-icon" alt="cc-icon" src="@/assets/cc_icon.svg"><img
-          v-for="(license, index) in onGetLicenseIcon(image.license)"
+    <img class="photo-license-icon" alt="cc-icon" src="@/assets/cc_icon.svg">
```
Comment (brenoferreira) [inline src/components/LicenseIcons.vue]: Missing a `/>` at the end here

File: src/components/LicenseIcons.vue [hunk 2]
```diff
@@ -4,12 +4,14 @@
...
+    <template v-for="(license, index) in onGetLicenseIcon(image.license)">
+      <img
           v-if="license" class="photo-license-icon"
           :alt="`${license} license icon`"
           :src="require(`@/assets/cc-${license.toLowerCase()}_icon.svg`)"
           :key="index">
```
Comment (brenoferreira) [inline src/components/LicenseIcons.vue]: Also missing a `/>` at the end here

**Comment by hemanth-hk** [comment]
Yup, I added `/>` at the end of the `img` tag, Please review

**Pull request merged**